chicago/templates/event.html

```diff
@@ -43,41 +43,46 @@ <h4>Agenda</h4>
         <h4>Attachments</h4>
         <p>
           {% for document in event.documents.all %}
-            <i class='fa fa-fw fa-file-text-o'></i> <a href="{{document.url}}">{{document.note}}</a><br />
+            <i class='fa fa-fw fa-file-text-o'></i> <a href="{{document.links.first.url}}">{{document.note}}</a><br />
           {% endfor %}
         </p>
       {% endif %}
-
+      <div class="modal-links">
+        <!-- View on legistar -->
+        {{ event|get_legistar_link|safe }}
+      </div>
     </div>
     <div class="col-sm-6">
       {% now "Y-m-d" as todays_date %}
-      {% if attendance_taken and todays_date > event.start_time|date:"Y-m-d" %}
-        <h4>Attendance: {{attendance_present}} Present, {{attendance_absent}} Absent</h4>
-        <table class="table">
-          <tr>
-            <th></th>
-            <th>Name</th>
-            <th>Ward</th>
-            <th class="text-center">Present</th>
-            <th class="text-center">Absent</th>
-          </tr>
-          {% for p in attendance %}
-            <tr class="{% if not p.attended %}danger{% endif %}">
-              <td>
-                <div class="thumbnail-square">
-                  <img src='{{p.attendee|get_person_headshot}}' alt='{{p.attendee.name}}' title='{{p.attendee.name}}' class='img-responsive img-thumbnail' />
-                </div>
-              </td>
-              <td><a href="{% url 'person' p.attendee.slug %}">{{p.attendee.name}}</a></td>
-              <td>{{ p.attendee.latest_council_seat }}</td>
-              <td class="text-center">{% if p.attended %}<i class="fa fa-fw fa-check"></i>{%endif%}</td>
-              <td class="text-center">{% if not p.attended %}<i class="fa fa-fw fa-times"></i>{%endif%}</td>
+      {% if todays_date > event.start_time|date:"Y-m-d" %}
```

Comment:
we can use the "event.status" field here, which might be safer as it will do the right thing for cancelled meetings.

Reply:
cool. looks like we can just show this when `event.status == 'passed'` https://puddle.bunkum.us/chicago_council-534ebda?sql=select+distinct+status+from+event